app/models/user.rb

```diff
@@ -1,5 +1,5 @@
 class User < ApplicationRecord
-  attr_accessor :remember_token, :activation_token
+  attr_accessor :remember_token, :activation_token, :reset_token
   before_save :downcase_email
   before_create :create_activation_digest
   USERS_PARAMS = %i(name email password password_confirmation).freeze
```

Comment:
please, sort such as the below picture
![image](https://user-images.githubusercontent.com/65896888/90859273-18eb9e80-e3b2-11ea-8c8e-c5cc092043ae.png)

Reply:
okay, I will try to arrange it now